lib/awful/completion.lua

```diff
@@ -154,10 +161,25 @@ function completion.shell(command, cur_pos, ncomp, shell)
         while true do
```

Comment:
$ /usr/bin/env bash -c 'compgen -A command tru'
    true
    trust
    truncate
    true

Damn ... Also note, with or without a directory `./true`, the previous command yields *two* `true`s. However the command I used in my previous post yields *three* `true`s if the directory exists. It's crazy how I didn't realize these inconsistencies before. Obviously bash / readline does much more post-processing than I was aware of.

To summarize: It seems we still need `sort -u`.